### DISCUSSION TITLE
Weird alignment in "Recently starred repositories" SVG

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#741] [Q&A] [answered]
One of the SVGs I display [on my profile README](https://github.com/Andre601/Andre601) seems to have some rather strange alignments in it, where 2 of the listed repositories are completely misaligned.

Some of the descriptions seem to also be cut off, but I'm not sure if that is a cause of the misalignment, because the date of when it was starred is correct.

Image:
![image](https://user-images.githubusercontent.com/11576465/148333664-6783fca9-13bd-41f7-b8d9-283e46943d05.png)

**Comment by lowlighter** [comment, marked as answer]
Will dig into it, it may be related to the github emojis (since there are not text per se as they're not "official" emojis)

**Reply by lowlighter** [reply]
This was fixed by #760 and will be available in next release

**Reply by Andre601** [reply]
Can confirm this. The SVG is fixed. Thank you!

![image](https://user-images.githubusercontent.com/11576465/150366780-5fe10947-c247-46c7-873c-60d0b4468471.png)
